src/box/relay.cc

```diff
@@ -329,6 +327,7 @@ relay_start(struct relay *relay, struct iostream *io, uint64_t sync,
 	relay->state = RELAY_FOLLOW;
 	relay->sent_raft_term = sent_raft_term;
 	relay->last_row_time = ev_monotonic_now(loop());
+	relay->tx_seen_time = ev_monotonic_now(loop());
```

Comment:
Calling ev_monotonic_now twice here, I'm not 100% sure, but it might be quite costly, we can call it once and asign it both to last_row_time and tx_seen_time here.

Reply:
AFAICS it's as cheap as `ev_now()`, because it returns time updated once per event loop iteration:
https://github.com/tarantool/tarantool/blob/5da1f6beb875260ae1ba2c5d13b8d5acd081033d/third_party/libev/ev.c#L2214-L2218

So I'd leave this code as is.

Reply:
Thanks for explanation!